modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc

```diff
@@ -17,18 +17,18 @@
 
 . In the *Administrator* perspective of the web console, navigate to *Operators* -> *Installed Operators* to verify that the {gitops-title} Operator is installed.
 . Navigate to the {rh-app-icon} menu -> *OpenShift GitOps* -> *Cluster Argo CD*. The login page of the Argo CD UI is displayed in a new window.
-. Obtain the password for the Argo CD instance:
+. Optional: To log in with your {product-title} credentials, ensure you are a user of the `cluster-admins` group and then select the `LOG IN VIA OPENSHIFT` option in the Argo CD user interface.
++
+[NOTE]
+====
```

Comment:
@reginapizza and @varshab1210 : Should we say `oc adm groups new cluster-admins admin` or just `oc adm groups new cluster-admins` ?

Reply:
cluster-admins I think

Reply:
@varshab1210 Can you please verify and confirm?

Reply:
**$ oc adm groups new cluster-admins** creates a **cluster-admins** group but with **$ oc adm groups new cluster-admins admin** it also creates an **admin** user

Reply:
Hi @Srivaralakshmi it should something like `$ oc adm groups new cluster-admins <user>`